app/assets/javascripts/datatables_defaults.js

```diff
@@ -0,0 +1,21 @@
+/* Set default DataTable options */
+/* https://datatables.net/examples/advanced_init/defaults.html */
+/* https://datatables.net/reference/option/ */
+
+$.extend( true, $.fn.dataTable.defaults, {
+  scrollY: '50vh',              // https://datatables.net/examples/basic_init/scroll_y_dynamic.html
+  responsive: true,             // https://datatables.net/extensions/responsive/priority
+  pagingType: "full_numbers",
+  retrieve: true,
+  pageLength: 100,
+  searching: false,
```

Comment:
Yes.  Would you like it back as a default, or do we want to add it for specific tables?